### ISSUE TITLE
compose is undefined

### ISSUE THREAD

**Issue by andresmosqueravtex** [PDMLab/docker-compose#161]
hi everyone.

I'm trying to use docker-compose to mount some "testing" containers to run integrations tests. I'm trying to run a simple test file:

```
import * as path from 'path'
import compose from 'docker-compose'


let dockerInstances = beforeAll( ()=>{
    return compose.upMany(['servicename'],{cwd:path.join(__dirname)+'/../'})
})

afterAll(()=>{
    dockerInstances.stop();
})

it('just a test',()=>{
    expect(true).toBe(false);
})
``` 

the thing is... i'm receiving a error indicating that compose is undefined:

`TypeError: Cannot read property 'upMany' of undefined
`

am i missing something?

**Comment by Steveb-p** [comment]
```typescript
import { compose } from 'docker-compose'
```

We're not using "default" export.

**Comment by Steveb-p** [comment]
I have no idea - it should work like this.

Check if you actually installed `docker-compose`, cause maybe you just forgot to `npm install`/`yarn install` it?

**Comment by andresmosqueravtex** [comment]
Hi there, yup, the package was installed. i managed to get it work updating the import syntax and deleting the "compiled" version of my test:

```
import * as path from 'path'
import * as compose from 'docker-compose'
import supertest from 'supertest'


beforeAll( ()=>{
    return compose.upMany(['service'],{cwd:path.join(__dirname)+'/../'})
})

afterAll(()=>{
    return compose.stop()
})

describe ('POST /AUTH',()=>{
    it('just a test',(done)=>{
        supertest(app.callback()).post('/auth').....
    })
});
```

**Comment by Steveb-p** [comment]
Ok, apparently I've completely forgot how importing works :sweat_smile: 

We're not doing the default export, that's why the `import composer from 'docker-compose'` didn't work.

For it to work we would need to additionally export all the functions we're defining in a
```typescript
export default {
    // ...functions
}
```
object. @AlexZeitler WDYT? I don't think there would be any side effects to such a re-export?

**Comment by AlexZeitler** [comment]
I wasn't sure so I made some experiments here and we should be safe:

https://github.com/AlexZeitler/typescript-exports

**Comment by AlexZeitler** [comment]
As of the currently released :package: `0.23.12` `import compose from 'docker-compose'` will work as well.
